.pre-commit-config.yaml

```
@@ -36,33 +46,17 @@ repos:
       - id: debug-statements
       - id: debug-statements
       - id: debug-statements
+  - repo: https://github.com/astral-sh/ruff-pre-commit
+    rev: v0.0.290
+    hooks:
+      - id: ruff
+        args: [--fix]
```

Don't we need this (according to the documentation: https://docs.astral.sh/ruff/usage/#pre-commit):

```suggestion
      args: [ --fix, --exit-non-zero-on-fix ]
```

---

Ah yeah I see that now.

---

Fixed